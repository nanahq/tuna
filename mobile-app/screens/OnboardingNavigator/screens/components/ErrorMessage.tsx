import {Text} from "react-native";
import {tailwind} from "@tailwind";

export function ErrorMessage ({error}: {error: string}): JSX.Element {
    return (
        <Text style={tailwind('text-red-600 text-xs pb-1')}>{error}</Text>
    )
}
