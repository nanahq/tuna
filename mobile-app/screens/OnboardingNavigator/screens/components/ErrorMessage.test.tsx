import {render} from '@testing-library/react-native'
import {ErrorMessage} from './ErrorMessage'


describe('<ErrorMessage />', () => {
    it('should match snapshot', () => {
        const rendered = render(<ErrorMessage error='Something went wrong' />)
        // @ts-ignore
        expect(rendered.toJSON()).toMatchSnapshot()
    })
})
