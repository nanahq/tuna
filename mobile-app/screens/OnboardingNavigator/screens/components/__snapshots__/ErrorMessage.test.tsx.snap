// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> should match snapshot 1`] = `
<Text
  style={
    {
      "color": "rgba(220, 38, 38, 1)",
      "fontSize": 12,
      "lineHeight": 16,
      "paddingBottom": 4,
    }
  }
>
  Something went wrong
</Text>
`;
